06-reactJs/components/form.js

```diff
@@ -0,0 +1,30 @@
+var React = require('react');
+var _ = require('lodash');
```

Comment:
I know this is work in progress and you might use ``lodash`` in the future, but have in mind that if you're not going to use it, you shouldn't require it. ;)